### ISSUE TITLE
cold intolerance

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9979]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Cold intolerance is an abnormal sensitivity to a cold environment or cold temperatures

PMID: 9571469 There are four types of symptoms on exposure to cold in the condition known as cold intolerance: pain/discomfort, stiffness, altered sensibility and colour change. They may occur in isolation or in any combination. Symptoms were present in at least 64% of 200 patients 24 months after hand injury. Pain was found to be the most troublesome symptom. Colour change in the digits was regarded by patients as the least troublesome symptom. Symptoms may begin immediately after injury, but are more likely to develop with the first onset of cold weather if there is a lag period between injury and onset. Pre-existing cold related symptoms are uniformly made worse by hand injury. A definition of the condition has been proposed and it has been suggested that this collection of symptoms be known as "Trauma Induced Cold Associated Symptoms".

PMID: 28336049 Clinical manifestations of hypothyroidism range from life threatening to no signs or symptoms. The most common symptoms in adults are fatigue, lethargy, ***cold intolerance***, weight gain, constipation, change in voice, and dry skin, but clinical presentation can differ with age and sex, among other factors

 PMID: 20301789 Allan-Herndon-Dudley syndrome (AHDS), an X-linked disorder, is characterized in males by neurologic findings (hypotonia and feeding difficulties in infancy, developmental delay / intellectual disability ranging from mild to profound) and later-onset pyramidal signs, extrapyramidal findings (dystonia, choreoathetosis, paroxysmal movement disorder, hypokinesia, masked facies), and seizures, often with drug resistance. Additional findings can include dysthyroidism (manifest as poor weight gain, reduced muscle mass, and variable ***cold intolerance***, sweating, elevated heart rate, and irritability) and pathognomonic thyroid test results.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

ORPHA:226298 Central congenital hypothyroidism

ORPHA:442  Congenital hypothyroidism

ORPHA:59 Allan-Herndon-Dudley syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000855
- label: Cold intolerance
- parents: [Heat intolerance (HP:0002046)](https://hpo.jax.org/app/browse/term/HP:0002046)
- pmid: 
- def: The inability to maintain a comfortable body temperature in a cold environment or cold temperatures.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9979
- synonyms:
